src/components/AddItemForm/AddItemForm.js

```diff
@@ -0,0 +1,50 @@
+import Image from "next/image";
+import { useEffect, useRef, useState } from "react";
+import { useAppContext } from "../../context/AppContext";
+
+export default function AddItemForm() {
+  const { todoDispatch } = useAppContext();
+  const [content, saveContent] = useState("");
```

Comment:
useState setter has a naming convention of adding **set** as a prefix. e.g., setContent